[2021-09-15 16:16:45,858] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: first_dag.first_func_test_execute 2021-09-15T16:16:43.424824+00:00 [queued]>
[2021-09-15 16:16:45,869] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: first_dag.first_func_test_execute 2021-09-15T16:16:43.424824+00:00 [queued]>
[2021-09-15 16:16:45,869] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-09-15 16:16:45,870] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2021-09-15 16:16:45,870] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-09-15 16:16:45,882] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): first_func_test_execute> on 2021-09-15T16:16:43.424824+00:00
[2021-09-15 16:16:45,885] {standard_task_runner.py:52} INFO - Started process 205 to run task
[2021-09-15 16:16:45,889] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'first_func_test_execute', '2021-09-15T16:16:43.424824+00:00', '--job-id', '7', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/mydags.py', '--cfg-path', '/tmp/tmp0q1vc5dv', '--error-file', '/tmp/tmpre2p4j9h']
[2021-09-15 16:16:45,890] {standard_task_runner.py:77} INFO - Job 7: Subtask first_func_test_execute
[2021-09-15 16:16:45,928] {logging_mixin.py:104} INFO - Running <TaskInstance: first_dag.first_func_test_execute 2021-09-15T16:16:43.424824+00:00 [running]> on host 51ebd8399503
[2021-09-15 16:16:45,972] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=first_func_test_execute
AIRFLOW_CTX_EXECUTION_DATE=2021-09-15T16:16:43.424824+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-09-15T16:16:43.424824+00:00
[2021-09-15 16:16:45,973] {logging_mixin.py:104} INFO - first_func_test_execute
[2021-09-15 16:16:45,986] {python.py:151} INFO - Done. Returned value was: None
[2021-09-15 16:16:45,994] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=first_func_test_execute, execution_date=20210915T161643, start_date=20210915T161645, end_date=20210915T161645
[2021-09-15 16:16:46,022] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-09-15 16:16:46,060] {local_task_job.py:149} INFO - Task exited with return code 0
